src/test/java/com/artipie/maven/MavenITCase.java

```diff
@@ -96,9 +95,10 @@ void init() throws Exception {
             new RepoConfigYaml("maven").withFileStorage(this.tmp.resolve("repos"))
         );
         this.port = this.server.start();
-        final Path setting = this.tmp.resolve("settings.xml");
-        setting.toFile().createNewFile();
-        Files.write(setting, this.settings());
+        Files.write(
+            this.tmp.resolve("settings.xml"),
+            new MavenSettings(this.port).value()
+        );
```

Comment:
@genryxy all usages of `MavenSettings` class involve reading content and writing it to folder. I'd suggest to add method `writeTo` to the class that will write the file to specified folder. It will reduce code duplication and will make tests shorter